### ISSUE TITLE
TemplateEditor

### ISSUE THREAD

**Issue by lucaguindani** [catima/catima#291]
Cannot add field variables in the item type template editor.

```
Uncaught TypeError: Cannot read property 'invoke' of undefined
at HTMLUListElement.click (TemplateEditor.jsx:89)
at HTMLUListElement.dispatch (Control.Layers.js:345)
at HTMLUListElement.elemData.handle (Control.js:164)
```

**Comment by lucaguindani** [comment]
Summernote issue -> https://github.com/summernote/summernote/issues/3249

**Comment by lucaguindani** [comment]
Solved by #292
